docs/UserGuide.md

```diff
@@ -11,8 +11,109 @@ support engineers, and others can validate a cluster’s configuration against a
+- A published `metadata.yaml` identifies the set of compliant RDS CR objects. Any new deviation found after running `kubectl cluster compare` tool  shall be identified as a new deviation gap to be worked out by the Red Hat RDS compliance process.  
+- Any incoming customer/partner request to modify the metadata.yaml due to an identified deviation gap shall be brought to the Red Hat RDS compliance process.  RDS compliance process will analyze the impact of the new requested deviation and ultimately intend to bridge the gap between the compliant  and the new identified deviation.
+- An RDS is decomposed into multiple RDS `Parts` (e.g., DU-Reference, Core-reference). A `part` typically identifies a workload or a set of workloads.
+
+```yaml
+# Every part denotes a partner workload or set of workloads of the end-to-end solution
+ Parts:
+  - name: DU-Reference 
+    ---- here goes the DU-Reference Specificaiton ----
+  - name: 5G-Core-Reference
+    ---- here goes the 5G Core-Reference specification ----
+```
+
+- Each RDS part is decomposed into multiple RDS components.
+
+```yaml
+# Every component denotes a feature particular of the component
+Parts:
+  - name: DU-Reference
+Components:
+name: Accelerator
+       ---- Accelerator Specification ----
+---- more components ----
+```
+
+- An RDS component is decomposed into multiple reference CRs.
+
+```yaml
+# requiredTemplates contains all needed reference CRs to comply with Accelerator component
+Components:
+Name: Accelerator
+type: Optional
+requiredTemplates:
+  - AcceleratorsNS.yaml
+  - AcceleratorsOperatorStatus.yaml
+  - AcceleratorsOperGroup.yaml
+  - AcceleratorsSubscription.yaml
+  - SriovFecClusterConfig.yaml
+```
+
+- An RDS Custom Resource is put into exactly one template YAML file.
+
+```yaml
+# Contents of AcceleratorNS.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: vran-acceleration-operators
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+```
+
+- An RDS CR (located in a template file) belonging to a required RDS component could not be required.
+
+```yaml
+# E.g., SR-IOV is an RDS component required for DU RDS. However, not all SRIOV reference template files in the source-crs folder associated with SR-IOV components are required. This is especially true for the kind.yaml vs kind-SetSelector.yaml variants for many RDS components. In the example shown below it is up to the user to specify whether the CR of kind SriovNetworkNodePolicy is configured using either SriovNetworkNodePolicy.yaml or SriovNetworkNodePolicy-SetSelector.yaml:
+- name: Sriov
+        type: Required
+        requiredTemplates:
+#         - SriovNetworkNodePolicy-SetSelector.yaml
+          - SriovNetworkNodePolicy.yaml
+```
+
+- A Reference Design CR is mapped to an OpenShift cluster version. Reference Designs CRs may change (or not) from one Openshift cluster version to another version.
+
 ## Optional vs required
```

Comment:
Similar comments as before -- this content looks ideally suited for more downstream documentation focused on the Telco use case models.  I thought I heard this tool was going to be more generically suited, but I could be completely mistaken.

Could we adjust this to focus more on a "reference configuration", this obfuscating this to be generic for this audience and user set?

Reply:
ACK. Will go over the PR and `generalize` the content to not be telco specific